posthog/management/commands/start_temporal_worker.py

```diff
@@ -344,7 +344,7 @@ def shutdown_worker_on_signal(worker: Worker, sig: signal.Signals, loop: asyncio
         with asyncio.Runner() as runner:
             loop = runner.get_loop()
 
-            configure_logger(loop=loop, default_mode=settings.TEMPORAL_LOG_DEFAULT_MODE)
+            configure_logger(loop=loop, default_mode=settings.TEMPORAL_LOG_DEFAULT_MODE)  # type: ignore[arg-type]
```

Comment:
Since this is coming from an external input (env variable) there is no way to type hint it except by manually validating the value at runtime.

Validating would protect us against mistyping the value in our internal configuration, but it would mean spending cycles every single time the program runs. Given that configuration values do not change often, and the consequences of an invalid value are not big (default log behavior), I'm choosing to ignore the type hint here.